how2meet/ui/pages/events.py

```diff
@@ -4,12 +4,7 @@
 
 from nicegui import APIRouter, ui
 
-from how2meet.utils import (
-    delete_event_api,
-    get_event_api,
-    get_events_api,
-    post_event_api,
-)
+from how2meet.utils import APIClient as api
```

Comment:
So you're kind of using the class as a collection of static functions here, which I'm cool with. My point about using classes was let's not force ourselves to instantiate an instance of a "client" class to be able to call functions, which is not what's happening here. No suing will occur!

Reply:
> So you're kind of using the class as a collection of static functions here, which I'm cool with. My point about using classes was let's not force ourselves to instantiate an instance of a "client" class to be able to call functions, which is not what's happening here. No suing will occur!

Question: should the methods in APIClient use the `@staticmethod` instead of `@classmethod` ? If I'm honest I feel like I always screw that up.